### ISSUE TITLE
Bug with particle accelerator

### ISSUE THREAD

**Issue by Nici660** [Drillgon200/Hbm-s-Nuclear-Tech-GIT#149]
Well, either I'm doing something terribly wrong or this is a bug. Neither what I do (and i tried 4 times in creative), while constructing a particle accelerator, it always explode in the core and I checked everything around it, it can only be the core and the core was always right placed.

**Comment by Drillgon200** [comment]
Screenshot?

**Comment by Nici660** [comment]
![2021-10-03_12 36 14](https://user-images.githubusercontent.com/85819475/135750097-6050ef27-dde9-4fe9-8dbb-ab549268d3e4.png)
![2021-10-03_12 36 22](https://user-images.githubusercontent.com/85819475/135750100-3cac6caa-f4c7-4c73-a059-ea372e0584b9.png)
![2021-10-03_12 36 38](https://user-images.githubusercontent.com/85819475/135750102-08c7cbd3-047f-4dbc-82e0-21eedf6b6a82.png)
![2021-10-03_12 36 50](https://user-images.githubusercontent.com/85819475/135750105-0deabe87-153e-496d-8d69-aa0babc51525.png)
![2021-10-03_12 37 12](https://user-images.githubusercontent.com/85819475/135750109-e13e1490-9023-4f05-aa2e-7e0da950475a.png)
![2021-10-03_12 38 06](https://user-images.githubusercontent.com/85819475/135750113-2b85f9d8-8459-489e-bf24-7eccb3a69f8a.png)

This is just an example Build in creative, my survival one is bigger than that one.

**Comment by whatsapp2** [comment]
I have the same issue in my survival world, there's an explosion at the core, rebuild the accelerator from 0 and the explosion still occurs

**Comment by Boon91** [comment]
Same issue as above cores placed right just explodes as soon as you try to start it rebuilt like 6 times thinking maybe I built it wrong or something

**Comment by Drillgon200** [comment]
![2021-10-06_17 46 03](https://user-images.githubusercontent.com/50186362/136303050-06b2728c-d5f3-416c-a940-6475bc9987d9.png)
Just built this thing, seems to work ok. All the same, I'll look and see if any other values got changed. All the coil values were updated, so there's probably some other values in the hadron itself that weren't updated.

**Comment by UFFR** [comment]
The most likely non-bug related issue is that you're not supplying 
enough power. Use higher tier plugs or use more of what you're currently 
using, high tier coils are power hogs. Also use a 20Hz DNT transformer 
(the one made with 4000k magnets) to more quickly charge them. It's 
possible the failure will occur at another section of the hadron, 
meaning that the problem /was/ power related and you still need more.

-- 
UFFR

**Comment by Nici660** [comment]
No, i don’t think that it is the power supply, these are only schrabidic coils they don‘t even need one power plug on every segment and this is a full spark energy storage block. Also, the explosion occurs every time where the core is.

**Comment by UFFR** [comment]
Add more power anyway, it's a simple solution and then we can scratch it 
off if it doesn't work.

-- 
UFFR

**Comment by Drillgon200** [comment]
There were some values not updated in the class itself, and I didn't port the higher tier power plugs so it's only the equivalent of second highest tier. More specifically, while coils generally have a much higher value, I didn't update the part where particles no longer have a max charge, and HE is worth far more particle charge.
